PendingReleaseNotes.md

```diff
@@ -2,5 +2,7 @@
 
 ## Breaking Changes
 
+- Updating Ceph COSI driver images, this impact existing COSI `Buckets` and `BucketAccesses`,
+please update the `BucketClass` and `BucketAccessClass` for resolving refer [here](https://github.com/rook/rook/discussions/14297)
```

Comment:
There will be upgrade steps added to this issue?

Reply:
The discussion is sparse. Could you provide a more detailed set of instructions for users to follow along? 

Doesn't this also affect any BucketClaims and BucketAccesses that are created? What about those?

Reply:
@thotz Before we merge this, please add the upgrade steps to the issue you opened.

Reply:
@travisn I have update the steps in https://github.com/rook/rook/discussions/14297